### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by milo-i**
Footer component finalized

### CODE REVIEW

**Review by MatsHaby — APPROVED** [review]
Ser bra ut!

File: design/src/App.scss [hunk 1]
```diff
@@ -120,6 +120,164 @@ $ribbon-color: #9a0017;
...
+  height: 100vh;
+  background-color: rgba(0,0,0,0.5);
+  z-index: 100;
+}
+
+.form-section {
+  width: 100vw;
+  max-width: 100%;
+  position: relative;
+  height: 100vh;
```
Comment (postmodernistx) [inline design/src/App.scss:137]: If there's text content here, consider using `min-height` as a property instead. If someone has a very short browser window, the content will be clipped, which creates an unprofessional look :)

File: design/src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+ return (
+  <footer className="footer">
+   <div className="footer-item first">
+    <p>GLÖGGA AB</p>
+    <p>Glöggvägen 24<br />123 45 Glöggköping</p>
+    <p>010-11 22 333<br />kundservice@glogga.se</p>
+   </div>
+   <div className="footer-item second">
+    <a href="www.omoss.se">Om oss</a>
```
Comment (postmodernistx) [inline design/src/components/Footer.jsx]: Should the `href` attribute's value be replaced with `#`?

File: design/src/components/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+ return (
+  <div>
+   <div className="img-overlay">
+   </div>
+   <div className="form-section">
+    <div className="form-wrapper">
+     <h3>Beställ smakprov på årets glögg</h3>
+     <input type="text" placeholder='Namn' id="name" value={name} onChange={(e) => setName(e.target.value)} /><br />
+     <input type="text" placeholder='E-mail' id="email" value={email} onChange={(e) => setEmail(e.target.value)} /><br />
```
Comment (postmodernistx) [inline design/src/components/Form.jsx:19]: The input type for an email field could be set to `email` for better user experience

File: design/src/components/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+     <input type="text" placeholder='Adress' id="address" value={address} onChange={(e) => setAddress(e.target.value)} /><br />
+     <input type="text" placeholder='Telefonnummer' id="phone" value={phone} onChange={(e) => setPhone(e.target.value)} /><br />
```
Comment (postmodernistx) [inline design/src/components/Form.jsx:21]: This input type could be `tel`

**Review by postmodernistx — APPROVED** [review]
Good with the small image sizes 👍 

Change the input field types to increase usability.

**Pull request merged**